.gitignore

```diff
@@ -37,7 +37,9 @@ testem.log
 .local.env
 .local.env.prod
 .firebase
+.firebaserc
 firebase-debug.log
+.runtimeconfig.json
```

Comment:
This file stores configurations (i.e. secrets) for local emulator running functions, ...